airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/write/StreamLoader.kt

```diff
@@ -63,4 +67,9 @@ interface BatchAccumulator {
         throw NotImplementedError(
             "processRecords must be implemented if createBatchAccumulator is overridden"
         )
+
+    suspend fun processFilePart(file: DestinationFile, index: Long): Unit =
```

Comment:
Can we make this a separate accumulator? Since it's already created by a different method call?

Reply:
Will do.

Reply:
Done